src/Command/CreateArticle.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Command;
+
+class CreateArticle implements Command
```

Comment:
Mmh what is this `Command` interface? Is it really needed? A command is supposed to be a very basic DTO, with data read/write capabilities only. Plus, as you don’t implement any method in this command, my guess is that this interface is basically empty? I’d say that it’s a bad practice because empty interfaces are generally used to ensure some kind of «identity» for the implementing classes, when interfaces should actually ensure behavior contracts only (so: methods). If it’s the command bus you’re using that imposes this constraint, you should check if it’s absolutely mandatory, or maybe use another command bus, I don’t know…

Reply:
Yes you're right the `Command` is an empty interface. I thought it was a better practive to always depends on interfaces but I get your point. For 'dump' object like commands are, this is pointless. Thanks for the advice, I learned something :)